check.js

```diff
@@ -8,29 +8,35 @@ function auditOk(issues) {
 }
 
 function printHumanReadableReport(issues) {
-    if (auditOk(issues)) {
-        console.log("audit ok.");
-        return;
-    }
-    issues.forEach(issue => {
+    return new Promise((resolve) => {
```

Comment:
The more I look at the code, the more I think this fixing the issue is accidental. The promise is not really waiting for anything, buffers are flushed behind the scenes while Node.js runs another event loop cycle and if the output was long enough, it'd still end the process before all content is flushed.

Reply:
Or you did it just to make the function return a promise which was not necessary.

Reply:
I made it return a promise to chain it in the then chain line 52.
It does not need to be waited for as the output seems to be in small enough chunks but it felt worse to have two print functions and only one is async.